### DISCUSSION TITLE
How can new notes be created within a sub-folder with the same name?

### DISCUSSION THREAD

**Discussion by congnvp** [marcusolsson/obsidian-projects#768] [Help]
I've used projects for several month. But whenever I created a new note, I have to manually move it in to subfolders with the same name using folder notes. Is there any way to create it in subfolders automatically? My new note have embed ```folder-index-content``` in Folder Index plugin so whenever I move a note from folder to sub-folder, it have a little bit lagging.

**Comment by Acylation** [comment]
Does the project option `Location for new notes` help? This can be found in the three-dot dropdown besides the project selector. Subfolders are supported.

![image](https://github.com/marcusolsson/obsidian-projects/assets/73122375/0daf3096-9b2c-4438-aebc-493855116792)

**Reply by congnvp** [reply]
Not helped. It isn't supporting the name of the note. I have to store all related file in that folder, if the number of task increase to hundreds then it will be big problem. You can see an example here.
![image](https://github.com/marcusolsson/obsidian-projects/assets/35249105/9378806b-3982-450c-a8a3-875ceeeeb3ec)
